### ISSUE TITLE
Title of site is generic

### ISSUE THREAD

**Issue by dengeist** [thinkful-ei18/schedio-client#75]
**Affected views:** All 🆘 

**The problem:** Clear document titles for each view of your UI are helpful for _all_ users, to help them understand where on the Web, and where within a web application, they currently are. Are they on Github.com's homepage? Are they on a new issue page?

**The fix:** _At least_ set a base title of `Schedio` in the `index.html` file. Beyond that, work out a system for programmatically adding route titles. For instance, the Settings view might expose a title of `Settings | Schedio`. The sub-route is named first so the user does not have to read farther or listen longer to know where they are. Hint: Dan Abramov has made [a lib for this](https://github.com/gaearon/react-document-title). Because the programmatic change will involve new code, this should _not_ be done before Friday.

**Comment by evang522** [comment]
fixed with minimum solution
